pkg/pipeline/service.go

```diff
@@ -25,6 +25,15 @@ import (
 	"github.com/conduitio/conduit/pkg/foundation/metrics/measure"
 )
 
+type FailureEvent struct {
+	// ID is the ID of the pipeline which failed.
+	ID            string
+	ProvisionType ProvisionType
+	Cause         error
```

Comment:
`Cause` makes sense, but why not call it `Error`? I'm not that attached to it! just a suggestion

Reply:
Error sounds good as well. :+1: